modules/apps/portal-security/portal-security-iframe-sanitizer/build.gradle

```diff
@@ -1,4 +1,5 @@
 dependencies {
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
+	compileOnly group: "org.jsoup", name: "jsoup", version: "1.15.3"
```

Comment:
sorry for the noise, but a question:

Didn't we want to get rid of this lib?

Reply:
no.

In fact @adolfopa suggested me to use it

Reply:
:D 

perfect then!